src/components/User/Limits/components/Limits.Manager.vue

```diff
@@ -319,13 +310,39 @@
         }
         this.isPending = false
       },
+      async removeLimits (limits) {
+        this.isPending = true
 
+        if (limits.id === 0) {
+          this.isPending = false
+          return false
+        }
+        if (!await confirmAction({ title: 'Delete the limits?' })) return
+        try {
+          const operation = Sdk.base.ManageLimitsBuilder.removeLimits({
+            id: (limits.id).toString()
+          })
+          await Sdk.horizon.transactions.submitOperations(operation)
+          await this.getLimits()
+          this.$store.dispatch('SET_INFO', 'Limits removed')
+        } catch (e) {
+          console.error(e)
+          e.showMessage()
+        }
+        this.isPending = false
+      },
       async getAssets () {
         const response = await Sdk.horizon.assets.getAll()
         this.assets = response.data
       },
       async getAccountIdByEmail (email) {
```

Comment:
pls rename to `loadAccountIdByEmail`. `get` should return something, `load` is more likely fetches and assigns something

Despite it is out of scope of your changes some minor refactors are appreciated